spec/requests/api/tenant_quotas_spec.rb

```diff
@@ -0,0 +1,263 @@
+describe "tenant quotas API" do
+  let!(:root_tenant) { Tenant.seed }
+
+  it "can list all the quotas form a tenant" do
```

Comment:
Since these tests require an appropriate role for the user, they may be better placed under the context for that below